Source/WebInspectorUI/UserInterface/Views/AuditTestContentView.css

```diff
@@ -34,11 +34,11 @@
 }
 
 .content-view-container > .content-view.audit-test > header h1 {
-    font-size: 2em;
+    font-size: 2rem;
```

Comment:
do we know for sure that `2em` will always be equal to `2rem` in every circumstance here?

ditto with all other `em` to `rem` changes below

Reply:
Do we reference the editor's drafts or the working drafts? According to the [spec](https://www.w3.org/TR/css-values-4/#rem) rem is
equal to the computed value of the [em](https://www.w3.org/TR/css-values-4/#em) unit on the root element.

Reply:
`Command +/-` is implemented by the web inspector. That is why I changed my mind. It zooms in on the px.

Reply:
my question is more about this particular situation than the behavior of `em` vs `rem`

for example, consider

```html
<div id="a" style="font-size: 20px">
  <div id="b" style="font-size: 2em"></div>
</div>
```

currently the computed `font-size` of `#b` is `40px`, but if we changed the `font-size` of `#b` to be `2rem` instead it'd be `32px` (since the default root `font-size` is `16px`)

so my question was more about whether we're sure that the `font-size` of every parent of every element matching `content-view-container > .content-view.audit-test > header h1` is the same as the root `font-size`, or if there's a scenario where some parent sets a `font-size` that's different from the root `font-size` (which would cause `2em` to compute to a different value than `2rem`)?